Classes/Form/Field/Text.php

```diff
@@ -39,6 +40,11 @@ class Text extends Input implements FieldInterface
     /**
      * @var string
      */
+    protected $richtextConfiguration;
```

Comment:
No, we have to keep that empty there, as well as in the `registerArgument` call - otherwise we'd never know that we should fall back to the TS value.